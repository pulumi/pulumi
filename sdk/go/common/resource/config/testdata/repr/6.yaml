my:parent:
  nested:
    foo: 4.2
