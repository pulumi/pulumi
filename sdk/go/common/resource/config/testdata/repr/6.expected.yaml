decrypt:
  my:parent: '{"nested":{"foo":4.2}}'
paths:
  my:parent:
    value:
      nested:
        foo: 4.2
    string: '{"nested":{"foo":4.2}}'
    redacted: '{"nested":{"foo":4.2}}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAABB/4I+AAEGbmVzdGVkF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4IaAAEDZm9vB2Zsb2F0NjQICgD4zczMzMzMEEA=
    secure: false
    isObject: true
  my:parent.nested:
    value:
      foo: 4.2
    string: '{"foo":4.2}'
    redacted: '{"foo":4.2}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAd/4IaAAEDZm9vB2Zsb2F0NjQICgD4zczMzMzMEEA=
    secure: false
    isObject: true
  my:parent.nested.foo:
    value: "4.2"
    string: "4.2"
    redacted: "4.2"
    object: EBAABnN0cmluZwwFAAM0LjI=
    secure: false
    isObject: false
